packages/kbn-test/src/auth/helper.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import * as fs from 'fs';
+import { User } from './session_manager';
+
+export const getProjectType = (serverArgs: string[]) => {
```

Comment:
__question:__ I see this function isn't used anywhere yet, is it just a leftover or is it reserved for some future use?